src/expr/core/src/expr/build.rs

```diff
@@ -110,6 +110,16 @@ where
                 // Dedicated types
                 E::All | E::Some => SomeAllExpression::build_boxed(prost, build_child),
 
+                // Iceberg partition transform functions
+                // # TODO
+                // Move to general types
+                E::IcebergBucket => external::iceberg::Bucket::build_boxed(prost, build_child),
```

Comment:
Once written in `#[build_function(...)]`, these matching arms are no longer needed.